src/oca/python/Makefile

```diff
@@ -15,6 +15,7 @@
 
 # Use full path to ensure virtualenv compatibility
 PYTHON = $(shell which python)
```

Comment:
Does it make sense to be also version specific for Python 2? (`which python2`)
(although nowadays systems still use symlink to python->python2 by default)